model/project_parser.go

```diff
@@ -987,6 +990,19 @@ func evaluateTaskUnits(tse *taskSelectorEvaluator, tgse *tagSelectorEvaluator, v
 	return tasks, groups, evalErrs
 }
 
+func shouldRunOnContainer(task ProjectTask, containers []Container) bool {
```

Comment:
Does this take into account containers from the project settings?

Instead of adding another config setting for users, can we instead resolve the running on distro vs. containers question in project validation + `createOneTask`?

Reply:
Containers themselves arent defined on project settings, just the container resource aliases that are referenced by 'size' on containers in the YAML.  They will be picked up from the project ref when they are needed later.